docs/DeveloperGuide.md

```diff
@@ -293,20 +293,23 @@ The following sequence diagram shows how deleting a session works:
 ![DeleteSessionSequenceDiagram](images/shion/DeleteSessionSequenceDiagram.png)
 
 It shares the same design considerations as what is mentioned in Add Session Feature.
+Note that: this feature is shared below, in deletion of an entire recurring session.
 
 ### Recurring Session
 Recurring Session is facilitated by the `RecurringSession` class which stores specific details of
 a tuition session with one student, that recurs. Similar to `Session` each recurring session is composed within a `Student`'s list of sessions,
 and a `Student` can have multiple `RecurringSession`s. <br>
 The class `RecurringSession` inherits from `Session`, albeit with additional properties `Interval` and `LastSessionDate`.
 This is because a recurring session can be defined a session that recurs, by given intervals, and up till a last date of session. <br>
-
+![SessionAndRecurringSessionClassDiagram](images/shion/SessionAndRecurringClassDiagram.png) <br>
 Note that:
```

Comment:
Same as comment above!

Reply:
This might be a slightly different kind of note (more like idk how to code within a div) so i think ill keep it as it is here. But thanks for comments on the other "note"